node/pkg/common/chainlock.go

```diff
@@ -15,8 +15,54 @@ import (
 	"go.uber.org/zap"
 )
 
-const HashLength = 32
-const AddressLength = 32
+const (
+	HashLength    = 32
+	AddressLength = 32
+
+	// The minimum size of a marshaled message publication. It is the sum of the sizes of each of
+	// the fields plus length information for fields with variable lengths (TxID and Payload).
+	marshaledMsgSizeMin = 1 + // TxID length (uint8)
+		20 + // TxID ([]byte), minimum length of 20 bytes
```

Comment:
Any reason not to use `HashLength` here? I think the TxID is either an `ethCommon.Hash` (32 bytes) or `solana.Signature` (64 bytes).

Reply:
thanks that makes sense. HashLength is 32 though, so how does that work with Solana.Signatures?

Reply:
Actually, reading farther down in the code, it looks like you add the txID length to this min, so should it even be included here?

Reply:
Yeah, the TxID is no longer fixed length. It is currently either 32 or 64, but I guess in the future it might be a different length. I guess the min it should be now is 32. But since you add the exact length in later, I'm not sure why you need to add anything here? (And why 20 anyway?)

Bottom line, can you just delete line 25 because you add in the lenght below?

Reply:
The 20 was to represent an EVM address but I realize that was a mitskae.

Line 25 is the timestamp, did you mean a different line?

The goal with this context is to do a quick failure for byte slices less than this length.